### ISSUE TITLE
Issue with the arp example

### ISSUE THREAD

**Issue by rfuentess** [Montimage/MMT_Security#3]
Hello, 

With the current  state of MMT Core, the example  of arp_probe seem to be outdated. 

The line 74: `#include "tcpip/mmt_tcpip_protocols.h"` Is not legacy?

Is vital to the line 184: ` mmt = mmt_init_handler( PROTO_SLL, 0, errbuf );` due PROTO_SLL is defined there (value 625). But, in debian installer said library is not included.

**Comment by hn-nguyen** [comment]
No, PROTO_SLL still there, in `/usr/local/include/mmt/tcpip/mmt_tcpip_protocols.h`, line 6389.

**Comment by rfuentess** [comment]
Hi @hn-nguyen, 

One question, with the debian package installer that were provided to me (Sep 8),   said library is not included. I have it, but  that was from before using  the debian package.

**Comment by hn-nguyen** [comment]
Can you check the installer I gave you on 07 Dec ?

If you still have pb, you can come to Montimage next Monday, 08 Feb (morning or after 15h as you want). We will see the problem together.
